internal/indexcoord/task.go

```diff
@@ -126,6 +126,10 @@ func (cit *CreateIndexTask) OnEnqueue() error {
 func (cit *CreateIndexTask) PreExecute(ctx context.Context) error {
 	log.Info("IndexCoord CreateIndexTask PreExecute", zap.Int64("collectionID", cit.req.CollectionID),
 		zap.Int64("fieldID", cit.req.FieldID), zap.String("indexName", cit.req.IndexName))
+	// TODO: check index type is disk index.
+	if GetEngineType(cit.req.GetIndexParams()) == diskAnnIndex && !cit.indexCoordClient.nodeManager.ClientSupportDisk() {
```

Comment:
EngineType is confusing. 
At present, it is recommended to directly judge whether it is a disk index.